pallets/admin-utils/src/lib.rs

```diff
@@ -1825,7 +1825,13 @@ pub mod pallet {
         /// Sets the admin freeze window length (in blocks) at the end of a tempo.
         /// Only callable by root.
         #[pallet::call_index(74)]
-        #[pallet::weight((0, DispatchClass::Operational, Pays::No))]
+        #[pallet::weight((
+			Weight::from_parts(5_771_000, 0)
+				.saturating_add(<T as frame_system::Config>::DbWeight::get().reads(0_u64))
+				.saturating_add(<T as frame_system::Config>::DbWeight::get().writes(1_u64)),
+			DispatchClass::Operational,
+			Pays::No
```

Comment:
Do we still want specific DispatchClass and Pays::No on admin utils extrinsic of should it be uniform?